### ISSUE TITLE
Fullscreen content

### ISSUE THREAD

**Issue by Donkey-Doug** [Tunous/Dawn#259]
Videos, photos and gifs are not shown fullscreen. As far as I am aware, it is not possible to do so.

**Comment by AbsurdlySuspicious** [comment]
Viewer that opens on click on media is not enough fullscreen?

**Comment by Donkey-Doug** [comment]
Here are some examples of what it looks like on my phone:

![Screenshot_2020-06-16-16-16-02-124_me thanel dank](https://user-images.githubusercontent.com/50382414/84786255-ccb86280-afec-11ea-9130-eaec59d7521d.jpg)
![Screenshot_2020-06-16-16-16-22-675_me thanel dank](https://user-images.githubusercontent.com/50382414/84786244-c924db80-afec-11ea-8366-21381b3d94fa.jpg)
![Screenshot_2020-06-16-16-16-18-644_me thanel dank](https://user-images.githubusercontent.com/50382414/84786246-c9bd7200-afec-11ea-9936-0c5f1d14fbc3.jpg)
![Screenshot_2020-06-16-16-16-06-087_me thanel dank](https://user-images.githubusercontent.com/50382414/84786250-cb873580-afec-11ea-8073-6b3b98dcaa0a.jpg)

**Comment by AbsurdlySuspicious** [comment]
They're fullscreen. Bottom controls can be hidden if you tap again on opened media. Missing ability to hide *video* controls is the only issue I can think of

**Comment by Donkey-Doug** [comment]
I am sorry, but they are not fullscreen. This is what fullscreen looks like on my phone (which has a non 16:9 aspect ratio):

Without controls:
![Screenshot_2020-06-16-16-26-19-539_org schabi newpipe](https://user-images.githubusercontent.com/50382414/84787416-3dac4a00-afee-11ea-8568-9adb08e8eb04.jpg)

With controls:
![Screenshot_2020-06-16-16-27-49-141_org schabi newpipe](https://user-images.githubusercontent.com/50382414/84787529-62082680-afee-11ea-8efb-a5129219db20.jpg)

**Comment by Donkey-Doug** [comment]
> And photos are sizeable in this mode, of course.

Nope, can only zoom in. There is no way to show a photo fullscreen in landscape mode, because I cannot zoom out.

**Comment by Donkey-Doug** [comment]
> Missing ability to hide _video_ controls is the only issue I can think of.

As well as the download, share, open in browser and HD buttons. All of this should be placed over the video content and hidden after a second of inactivity.

**Comment by AbsurdlySuspicious** [comment]
> Nope, can only zoom in. There is no way to show a photo fullscreen in landscape mode, because I cannot zoom out.

> As well as the download, share, open in browser and HD buttons. All of this should be placed over the video content and hidden after a second of inactivity.

These are two separate issues. Please, report them properly with describing titles, so it'd be easier to pick these issues from list for someone who want to work on this

**Comment by Tunous** [comment]
I can see that indeed we can toggle controls for videos but the video itself doesn’t resize and the progress bar is visible. That should be fixed. Lets use this issue to handle this situation.

The problem with image zoom is indeed worthy of its own issue.

**Comment by Tunous** [comment]
Moved the other one to https://github.com/Tunous/Dawn/issues/260
